README.md

```diff
@@ -1,58 +1,173 @@
+- *Search*
+  - zola gets ![ehh] because search engine is disabled by default in some languages, as explained [in the docs](https://www.getzola.org/documentation/content/multilingual/#configuration)
+
+### Content library
+
+|                                 | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
+|:--------------------------------|:------:|:------:|:------:|:-------:|
 | Aliases                         | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Pagination                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Custom taxonomies               | ![yes] | ![no]  | ![yes] | ![no]   |
-| Search                          | ![yes] | ![no]  | ![no]  | ![yes]  |
-| Data files                      | ![yes] | ![yes] | ![yes] | ![no]   |
-| LiveReload                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Netlify support                 | ![yes] | ![no]  | ![yes] | ![no]   |
-| Vercel support                  | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Assets co-location              | [![ehh]](https://www.getzola.org/documentation/content/overview/#asset-colocation) | ![yes] | ![yes] | ![yes]  |
+| HTML shortcodes                 | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) | ![no]  | ![yes] | ![yes]  |
+| Format-specific shortcodes      | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) |   ?    |    ?   |    ?    |
+| Internal links                  | [![ehh]](https://www.getzola.org/documentation/content/linking/) | ![no]  | ![yes] | ![yes]  |
+| Warn or error for broken links  | [![no]](https://github.com/getzola/zola/issues/977#issuecomment-759725671)  | ![no]  | [![yes]](https://gohugo.io/content-management/cross-references/) | ? |
+| Custom taxonomies               | [![yes]](https://www.getzola.org/documentation/content/taxonomies/) | ![no]  | ![yes] | ![no]   |
+| Multiple static mounts          | [![no]](https://github.com/getzola/zola/issues/499) | ![no] | [![yes]](https://gohugo.io/hugo-modules/configuration/#module-config-mounts) | ? |
+
+**Notes:**
+
+- *Assets co-location*
+  - zola receives ![ehh] because assets cannot be shared across pages/sections, despite living in the same folder ([discussion on the forum](https://zola.discourse.group/t/reusing-markdown-docs-from-github-repo-in-zola-site/776))
+- *Internal links*
+  - zola receives ![ehh] because so there is currently no reliable way to reference static assets from the content pages (though a shortcode can do it very easily), which is a limitation that only affects builds for a subfolder (see discussion about [path unification](https://github.com/getzola/zola/issues/977)) ; sites built for the webroot of a domain are unaffected by this limitation
```

Comment:
The Hugo link only refers to content links?

Reply:
To be clear, i'm not saying internal links are broken. I'm saying that common patterns that work with `siteroot = webroot` [doesn't work](https://zola.discourse.group/t/relative-path-for-my-website/395) with `siteroot = webroot/foobar` (yet) which people should be aware of. So you have to take extra steps (eg. a shortcode using get_url) in those situations.

hugo has really good support for relative URLs including an option (relativeURLs) that will force all URLs to be relative to the site root. I'm honestly this way is the best for zola (needs more thought about path unification), but i believe people making a site should be aware of this limitation. I'll try to make it more clear!